packages/docs/docs/dev/getting-started.mdx

```diff
@@ -27,17 +27,17 @@ To use it, you must also install the [`fuels`](https://www.npmjs.com/package/fue
 ### Installation
 
 ```bash
-npm install fuels @fuel-wallet/sdk
+npm install fuels @fuel-wallet/sdk @fuel-wallet/react
 ```
 
 ### Usage
 
-To use the SDK in your TypeScript project, add the following line to your tsconfig.json file:
+To use the SDK and the React Hooks in your TypeScript project, add the following line to your tsconfig.json file:
 
 ```json
 {
   "compilerOptions": {
-    "types": ["@fuel-wallet/sdk"]
+    "types": ["@fuel-wallet/sdk", "@fuel-wallet/react"]
```

Comment:
This should not be required when using `@fuel-wallet/react` the types are auto-imported.